app/main.py

```diff
@@ -1,8 +1,52 @@
 class Car:
-    # write your code here
+    def __init__(
+            self,
+            comfort_class: int == 1,
+            clean_mark: int == 1,
+            brand: str
+    ) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
     pass
 
 
 class CarWashStation:
-    # write your code here
+    def __init__(
+            self,
+            distance_from_city_center: float == 1.0,
+            clean_power: int == 1,
+            average_rating: float == 1.0,
+            count_of_ratings: int
+    ) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = round(average_rating, 1)
+        self.count_of_ratings = count_of_ratings
+
+    def calculate_washing_price(self, car: Car) -> float:
+        price = (car.comfort_class
+                 * (self.clean_power - car.clean_mark)
+                 * self.average_rating
+                 / self.distance_from_city_center)
+        return round(price, 1)
+
+    def wash_single_car(self, car: Car) -> None:
+        if self.clean_power > car.clean_mark:
+            car.clean_mark = self.clean_power
+
+    def rate_service(self, rate: float) -> None:
+        total_rating = self.average_rating * self.count_of_ratings
+        self.count_of_ratings += 1
+        self.average_rating = round((total_rating + rate)
+                                    / self.count_of_ratings, 1)
+
+    def serve_cars(self, cars: list) -> float:
```

Comment:
```suggestion
    def serve_cars(self, cars: list[Car]) -> float:
```
Add inner type annotations